docs/References.bib

```diff
@@ -1904,6 +1930,22 @@ @article{Porth2016rfi
   SLACcitation   = "%%CITATION = ARXIV:1611.09720;%%"
 }
 
+@article{Renkhoff2023,
+  author        = {Renkhoff, Sarah and Cors, Daniela and Hilditch, David and
+                   Br\"ugmann, Bernd},
+  title         = {Adaptive {hp} refinement for spectral elements in numerical
+                   relativity}},
```

Comment:
Extra `}`? (I did not actually look at the PR, just noticed this one thing)

Reply:
Fixed (don't know why the docs rendering worked)